core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -3226,15 +3226,12 @@ class WorkspaceService(protected val ctx: RawlsRequestContext,
     }
 
     for {
-      _ <- traceDBIOWithParent("requireCreateWorkspaceAccess", parentContext) { childContext =>
-        DBIO.from(requireCreateWorkspaceAction(billingProject.projectName, childContext))
-      }
+      _ <- failIfWorkspaceExists(workspaceRequest.toWorkspaceName)
```

Comment:
moved this here as it needs to be in the same transaction that the workspace record is being created in.